experiments/coredns/pod_delete/pod_delete_k8s_job.yml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: pod-delete-
+  namespace: %CHAOS_NAMESPACE%
+spec:
+  template:
+    metadata:
+      labels:
+        experiment: pod-delete
+    spec:
+      # Placeholder that is updated by the executor for automated runs
+      # Provide appropriate SA (with desired permissions) if executed manually
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: imrajdas/ansible-runner:ci
+        imagePullPolicy: Always
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          # provide application namespace
+          - name: APP_NAMESPACE
+            value: 'kube-system'
+
+          # provide application labels
+          - name: APP_LABEL
+            value: 'k8s-app=kube-dns'
```

Comment:
please remove the entries from all env variables

Reply:
These are the permanent env variable for the coredns pod

Reply:
@rajdas98  Namespace is okay, But label may vary then how you are updating the label.
cc @ksatchit

Reply:
Better to keep it empty

Reply:
I checked that every DNS component of k8s have the same namespace

Reply:
Okay!! Have you checked for all platform or only GKE?

Reply:
Yes, checked with GCP
```
kube-dns-79868f54c5-z77nj                                      0/4     ContainerCreating   0          42s   k8s-app=kube-dns,pod-template-hash=79868f54c5
```